public/app/plugins/datasource/cloudwatch/components/MetricStatEditor/MetricStatEditor.test.tsx

```diff
@@ -87,4 +87,17 @@ describe('MetricStatEditor', () => {
       expect(screen.queryByText('Match exact')).toBeNull();
     });
   });
+
+  describe('match exact', () => {
+    it('should be checked when value is true', () => {
+      render(<MetricStatEditor {...props} disableExpressions={false} />);
+      expect(screen.getByLabelText('Match exact - optional')).toBeChecked();
+    });
+
+    it('should be unchecked when value is false', () => {
+      props.query.matchExact = false;
```

Comment:
This is changing the property affecting other tests. You need to make a copy of the props.

Reply:
Yes, true. Will fix